wire-ios/Wire-iOS Share Extension/UnsentSendable.swift

```diff
@@ -291,11 +287,22 @@ class UnsentFileSendable: UnsentSendableBase, UnsentSendable {
             }
 
             if let data = data as? Data {
+                guard data.count <= AccountManager.fileSizeLimitInBytes else {
+                    self?.error = .fileSizeTooBig
+                    return completion()
+                }
+
                 self?.prepareForSending(withUTI: UTIString,
                                         name: name,
                                         data: data,
                                         completion: prepareColsure)
             } else if let dataURL = data as? URL {
+
+                guard dataURL.fileSize ?? .max <= AccountManager.fileSizeLimitInBytes else {
```

Comment:
@johnxnguyen Could we do like 
```
guard let fileSize = dataURL.fileSize, 
           fileSize  <= AccountManager.fileSizeLimitInBytes else {
}
```
?
If `fileSize` is nil we send the `.fileSizeTooBig` error.

Reply:
If we don't know the file size then we can't assume that it's smaller than the limit, so I think it's fine to show the error.